modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -18,6 +18,9 @@ any questions.
 
 - [Codemod](#codemod)
   - [Instructions](#instructions)
+- [Component Updates](#component-updates)
+  - [Styling API and CSS Tokens](#styling-api-and-css-tokens)
+  - [Skeleton](#skeleton)
```

Comment:
```suggestion
```
I think you can remove skeleton, we only add it here if we changed any API stuff, otherwise we lump it into style refactor!